.changeset/flat-starfishes-draw.md

```diff
@@ -0,0 +1,5 @@
+---
+"@wso2is/admin.branding.v1": patch
+---
```

Comment:
Let's update console as well to a new patch version.

Reply:
done!